tools/hangar/__snapshots__/test_corpus/sdk_tests/bucket/events.w_compile_tf-aws.md

```diff
@@ -1,5 +1,48 @@
 # [events.w](../../../../../../examples/tests/sdk_tests/bucket/events.w) | compile | tf-aws
 
+## CheckHitCountOptions.Struct.js
+```js
+module.exports = function(stdStruct, fromInline) {
+  class CheckHitCountOptions {
+    static _schema = {
+      id: "/CheckHitCountOptions",
+      type: "object",
+      properties: {
+        key: { type: "string" },
+        type: { type: "string" },
+        source: { "$ref": "/Source" },
+        count: { type: "number" },
+      },
+      required: [
+        "key",
+        "type",
+        "source",
+        "count",
+      ]
+    }
+    static _getDependencies() {
+      return {
+        "/Source": require("./Source.Struct.js")()._schema,
+        ...require("./Source.Struct.js")()._getDependencies(),
+      }
+    }
+    static _validate(obj) {
```

Comment:
Do we really need this function here? Seems like all structs will have the exact same validate method, which implies it should be under the `std.Struct` type.

Reply:
Its gone in its place is a `fromJson` method that just makes a call to std.Struct and passes in the self contained schema